src/errors/test-run/templates.js

```diff
@@ -207,5 +207,9 @@ export default {
         Action "${err.selectorName}" argument error:
 
         ${escapeHtml(err.errMsg)}
+    `),
+
+    [TYPE.requestedSizeExceedsMaximumSize]: err => markup(err, `
+        The requested size exceeds the available maximum size.
```

Comment:
We need better description for this one: we should describe that on MacOS window can't be bigger than screen size, so you can't resize to given dimensions